extensions/typescript-language-features/src/typescriptServiceClient.ts

```diff
@@ -564,27 +564,30 @@ export default class TypeScriptServiceClient extends Disposable implements IType
 		return this.executeImpl(command, args, {
 			isAsync: false,
 			token,
-			expectsResult: true
+			expectsResult: true,
+			lowPriority: true
```

Comment:
This will mark pretty much every request as low priority. You need to pass in the low priority flag to this function

Reply:
Done.